Poker.BE/Poker.BE.Service/Modules/Results/LogoutResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Poker.BE.Service.Modules.Results
+{
+	public class LogoutResult : IResult
+	{
+		public int User { get; internal set; }
```

Comment:
lol that's what happens when u let visual studio automatically generate functions